src/libfdt-bindings/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "libfdt-bindings"
+version = "0.1.0"
+authors = ["Amazon Firecracker team <firecracker-devel@amazon.com>"]
+edition = "2018"
+build = "build.rs"
```

Comment:
I'd prefer we only build this under `aarch64`. Either with an arch gate at the crate level (the crate wouldn't even exist on other archs), or with a code-level gate (crate exists but is empty).

Reply:
I couldn't find a way to add an arch gate at the crate level, but there are code-level gates. Also I added a gate for the dependencies.